.github/workflows/release.yml

```diff
@@ -1,12 +1,8 @@
-name: Perform a Release
+name: Publish
```

Comment:
Missed this in the other one but maybe this file should be named `publish.yml` now?